packages/components/constraints/src/helpers.js

```diff
@@ -0,0 +1,18 @@
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const getMaxPropTokenValue = (max) => {
+  return vars[`constraint${max}`] ?? null;
```

Comment:
Not necessary, it would be implicitly `undefined`

```suggestion
  return vars[`constraint${max}`];
```

Reply:
point was to ensure `null` as return fallback

Reply:
Either `null` or `undefined`, it doesn't actually make a difference. So I'll simplify and leave it undefined :)